core-addon/FirefoxPrivilegedApi.schema.json

```diff
@@ -2,6 +2,13 @@
   {
     "namespace": "firefoxPrivilegedApi",
     "description": "experimental API extensions for exposing privileged Firefox APIs",
+    "events": [
```

Comment:
ah! TIL! I'm not an expert of this privilege API exposure. Let me bring in somebody else :)

Reply:
Hey @rpl, would you kindly review the changes to this "experimental API"?

Reply:
Thanks :) FTR the `EventManager` thing is great :)